dotnet/src/dotnetframework/GxClasses/Helpers/HttpHelper.cs

```diff
@@ -467,7 +467,8 @@ public static void AddHeader(this HttpResponse response, string name, string val
 
 		public static void Write(this HttpResponse response, string value)
 		{
-			response.WriteAsync(value).Wait();
+			//response.WriteAsync(value).Wait();
+			response.Body.Write(Encoding.UTF8.GetBytes(value));
```

Comment:
I think @cmurialdo told the other day that we should use Async if possible. Maybe we can go back to use AsyncWrite instead of Sync().

Reply:
We changed it because it didn't work with the previous code. We should check it again.